### PR TITLE
Windows otel

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/windows-opetelemetry.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+### Step 1: Set up Collector
+
+{@import ../../../reuse/apps/opentelemetry/set-up-collector.md}
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/PCI-Compliance-For-Linux/OpenTelemetry/Linux-Collector.png' alt="Linux-Collector" style={{border: '1px solid black'}} />
+
+### Step 2: Configure integration
+
+In this step, you will configure the yaml file required for Windows event logs and metrics Collection.
```
Comment (jpipkin1) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/windows-opetelemetry.md:48]:
```suggestion
In this step, you will configure the YAML file required for Windows event logs and metrics Collection.
```

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/windows-opetelemetry.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+### Critical Events​
+
+The **WWindows - Security Monitoring - Critical Events** dashboard provides analysis of critical security events related to services stopped, audit logs tampered, and logging ingestion delays.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Windows-Cloud-Security-Monitoring-and-Analytics/OpenTelemetry/Windows-Security-Monitoring-Critical-Events.png' style={{border: '1px solid black'}} alt="Windows-Security-Monitoring-Critical-Events" />
+
+### Inventory​
+
+The **Windows - Security Monitoring - Inventory** dashboard helps you to monitor windows events provided by computer, channel, and provider. This dashboard also provides sdditional information on computer reboots.
```
Comment (jpipkin1) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/windows-opetelemetry.md:254]:
```suggestion
The **Windows - Security Monitoring - Inventory** dashboard helps you to monitor windows events provided by computer, channel, and provider. This dashboard also provides additional information on computer reboots.
```

**Comment by JV0812** [comment]
Thanks John. I have incorporated your comments.

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/windows-opetelemetry.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
+---
+id: windows-opentelemetry
```
Comment (kimsauce) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/windows-opetelemetry.md:2]: typo 

Screenshot 2023-07-25 at 2 37 59 PM

**Review by kimsauce — DISMISSED** [review]
typo in doc title

**Pull request merged**